sample/allofhands_hawaii2016/session_types

```diff
@@ -0,0 +1,73 @@
+[
+
+        {   
+            "id": 1, 
+            "length": null, 
+            "name": "Welcome Reception"
+        },
+        {
+            "id": 2, 
+            "length": null, 
+            "name": "Breakfast"
+        }, 
+        {
+            "id": 3, 
+            "length": null, 
+            "name": "Lunch"
+        },
+        {
+            "id": 4, 
+            "length": null, 
+            "name": "Team Time"
+        }, 
+        {
+            "id": 5, 
+            "length": null, 
+            "name": "Team Time"
```

Comment:
please review on how to make session_types. they should have a length as we need to show what this type represents. Also session_types with same name dont make sense here.